examples/wallets-demo/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@rango-dev/provider-all": "^0.28.1-next.2",
-    "@rango-dev/ui": "^0.29.1-next.13",
+    "@rango-dev/ui": "^0.29.1-next.14",
```

Comment:
As a rule, the package version should not be changed.
I think that if you pull the **next** branch, it will be fixed

Reply:
fixed